libs/application/templates/social-insurance-administration/death-benefits/src/lib/dataSchema.ts

```
@@ -0,0 +1,174 @@
+} from '@island.is/application/templates/social-insurance-administration-core/lib/constants'
+import { NO, YES } from '@island.is/application/types'
+import {
+  formatBankInfo,
+  validIBAN,
+  validSWIFT,
+} from '@island.is/application/templates/social-insurance-administration-core/lib/socialInsuranceAdministrationUtils'
+import * as kennitala from 'kennitala'
+
+const isValidPhoneNumber = (phoneNumber: string) => {
+  const phone = parsePhoneNumberFromString(phoneNumber, 'IS')
+  return phone && phone.isValid()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)